x-pack/plugin/security/qa/jwt-realm/build.gradle

```diff
@@ -43,6 +43,7 @@ testClusters.matching { it.name == 'javaRestTest' }.configureEach {
 
   // These realm settings are generated by JwtRealmGenerateTests
   setting 'xpack.security.authc.realms.jwt.jwt1.order', '1'
+  setting 'xpack.security.authc.realms.jwt.jwt1.token_type', 'id_token'
```

Comment:
should we randomize adding this to the setting to help test that it is defaulted if not explicitly defined ?

[here](https://github.com/elastic/elasticsearch/blob/main/x-pack/qa/multi-cluster-search-security/legacy-with-full-license/build.gradle#L15) is an example how to randomize in a build.gradle

Reply:
I didn't randomize because there is a `jwt3` realm which I didn't specify the `token_type` explicitly. So both variations are covered already. That said, it does not hurt to always randomize. So I added it. Note it will probably be changed again once access token support is added.